app/webpacker/components/CompetitionForm/FormSections/DuesEstimate.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useMemo } from 'react';
+import { Input, Modal } from 'semantic-ui-react';
+import useInputState from '../../../lib/hooks/useInputState';
+import { calculateDuesUrl } from '../../../lib/requests/routes.js.erb';
+import { currenciesData } from '../../../lib/wca-data.js.erb';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import Loading from '../../Requests/Loading';
+import Errored from '../../Requests/Errored';
+
+export default function DuesEstimate({
+  close, countryId, currencyCode, baseEntryFee, competitorLimit,
+}) {
+  const [competitorCount, setCompetitorCount] = useInputState(competitorLimit || 0);
+  const currencyInfo = useMemo(
+    () => (currenciesData.byIso[currencyCode] || currenciesData.byIso.USD),
+    [currencyCode],
+  );
+
+  const {
+    data, loading, error,
+  } = useLoadedData(calculateDuesUrl(countryId, currencyCode, baseEntryFee));
```

Comment:
Can we use `useQuery` here? `useLoadedData` has some issues and I think the plan is to replace it at some point anyway.

Reply:
Okay, replaced with `useQuery`. I actually moved the API call from `RegistrationFees` component.